### PR TITLE
Fix: Remove extra soft prompt asking for media permission on every app launch in macOS

### LINKED ISSUE #4750: [bug] Soft Prompts on launch
### Describe the bug

I am soft prompts asking microphone permission whenever I open Tauri app on macOS
Screenshot 2022-07-24 at 11 35 33 AM
.

Please check the attached image.

### Reproduction

_No response_

### Expected behavior

_No response_

### Platform and versions

```shell
Environment
  › OS: Mac OS 12.4.0 X64
  › Node.js: 16.13.2
  › npm: 8.9.0
  › pnpm: Not installed!
  › yarn: 1.22.17
  › rustup: 1.25.1
  › rustc: 1.62.0
  › cargo: 1.62.0
  › Rust toolchain: stable-x86_64-apple-darwin 

Packages
  › @tauri-apps/cli [NPM]: 1.0.5
  › @tauri-apps/api [NPM]: 1.0.2
  › tauri [RUST]: 1.0.4,
  › tauri-build [RUST]: 1.0.4,
  › tao [RUST]: 0.12.1,
  › wry [RUST]: 0.19.0,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../dist
  › devPath: http://localhost:5173/
  › framework: React

App directory structure
  ├─ dist
  ├─ node_modules
  ├─ public
  ├─ private
  ├─ .github
  ├─ src-tauri
  ├─ .git
  └─ src
```

### Stack trace

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by morajabi**
Fixes https://github.com/tauri-apps/tauri/issues/4750

Discord conversation: https://discord.com/channels/616186924390023171/731495028677148753/1020299521542082610

### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/wry/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: .changes/fix-double-dialog.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"wry": minor
+---
+
+Fix double permission dialog issue.
```
Comment (amrbashir) [inline .changes/fix-double-dialog.md]:
```suggestion
Fix double permission dialog on macOS 12+ and iOS 15+.
```

**Review by amrbashir — DISMISSED** [review]

File: .changes/fix-double-dialog.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"wry": minor
```
Comment (amrbashir) [inline .changes/fix-double-dialog.md]:
```suggestion
"wry": patch
```

**Review by amrbashir — APPROVED** [review]

**Comment by FabianLars** [comment]
Do we need some kind of macos version check? idk if this can cause panics otherwise 🤔

**Comment by amrbashir** [comment]
yeah this could probably panic, we need to test it on macOS 11 and below before merging

**Comment by lucasfernog** [comment]
I believe this won't crash on macOS below 12 since we're only adding a method, but I can double check.

Should we add a config for this instead of always allowing it?

**Comment by amrbashir** [comment]
Is there a security implications? otherwise it should be fine until someone asks for it to be configurable.

**Review by lucasfernog — APPROVED** [review]
Does not crash on macOS 10.15 so it should be fine :)

**Pull request merged**